### PR TITLE
feat(slack): Add analytics to slack actions

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Records actions taken on sentry unfurls and posted alerts.

### CODE REVIEW

File: src/sentry/integrations/slack/action_endpoint.py [hunk 1]
```diff
@@ -253,11 +254,19 @@ def post(self, request):
         # Handle interaction actions
         try:
             for action in action_list:
-                if action['name'] == 'status':
+                action_type = action['name']
+
+                analytics.backend.record(
```
Comment (mattrobenolt) [inline src/sentry/integrations/slack/action_endpoint.py]: Just use `analytics.record`.

File: src/sentry/integrations/slack/action_endpoint.py [hunk 2]
```diff
@@ -253,11 +254,19 @@ def post(self, request):
...
+                    'integrations.slack.action',
+                    action_type=action_type,
+                    actor_id=identity.user.id
```
Comment (mattrobenolt) [inline src/sentry/integrations/slack/action_endpoint.py]: Could this be `identity.user_id` to avoid accidentally querying?

**Review by mattrobenolt — COMMENTED** [review]
Are we sure we want to push these through analytics? What's the goal here? Or do you want these in metrics?

File: src/sentry/integrations/slack/action_endpoint.py (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/integrations/slack/action_endpoint.py]: 84da11c39

File: src/sentry/integrations/slack/action_endpoint.py (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/integrations/slack/action_endpoint.py]: dd5e34fe7

**Comment by ehfeng** [comment]
@mattrobenolt This is for product usage reports/graphs, so analytics is the right place for them.

**Review by ehfeng — APPROVED** [review]

**Pull request merged**